projects/packages/assets/changelog/update-i18n-loader-footer

```diff
@@ -0,0 +1,4 @@
+Significance: patch
+Type: changed
+
+i18n loader script: load script in the footer and defer it.
```

Comment:
Update this?

Reply:
Good point. 2c1de55532b8a1696de827c530c98b48d65b8549